src/js/pensions/config/form.js

```diff
@@ -184,6 +189,52 @@ const formConfig = {
               }
             }
           }
+        },
+        employmentHistory: {
+          title: 'Employment history',
+          path: 'employment/history',
+          depends: isUnder65,
+          uiSchema: {
+            'ui:description': 'Please tell us about all of your employment, including self-employment, from one year before you became disabled to the present.',
+            jobs: {
+              'ui:options': {
+                viewField: EmploymentField
+              },
+              items: {
+                employer: {
+                  'ui:title': 'Name of employer'
+                },
+                address: address.uiSchema('Address of employer'),
+                jobTitle: {
+                  'ui:title': 'Job title'
+                },
+                dateRange: dateRangeUI(),
+                daysMissed: {
+                  'ui:title': 'How many days lost to disability'
+                },
+                annualEarnings: {
+                  'ui:title': 'Total annual earnings'
+                }
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              jobs: {
+                type: 'array',
+                minItems: 1,
+                items: {
+                  type: 'object',
+                  required: ['address', 'employer', 'jobTitle', 'dateRange', 'daysMissed', 'annualEarnings'],
```

Comment:
@goldenmeanie I assume all this is right, but are all the fields from the original ticket required?